### ISSUE TITLE
Outdated PyMuPDF breaking dependencies

### ISSUE THREAD

**Issue by HongYurui** [dylanyang17/TsinghuaBookCrawler#9]
The `PyMyPDF` version in `requirements.txt` is `1.89.9`, which is currently outdated and such installations broke the dependencies  for other tools like `pdf2docx` and `pdf2pptx`:

```
$ pip install -r requirements.txt
...
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pdf2pptx 1.0.5 requires pymupdf==1.20.1, but you have pymupdf 1.18.9 which is incompatible.
pdf2docx 0.5.5 requires PyMuPDF>=1.19.0, but you have pymupdf 1.18.9 which is incompatible.
Successfully installed PyMuPDF-1.18.9 requests-2.25.1
```

It seems that `pymypdf==1.20.1` would not cause such problems but raising an exception:
```
Traceback (most recent call last):
  File "/home/hyr/Downloads/TsinghuaBookCrawler-2.1/main.py", line 115, in <module>
    img2pdf(imgs, pdf_path, quality)
  File "/home/hyr/Downloads/TsinghuaBookCrawler-2.1/img2pdf.py", line 31, in img2pdf
    pdfbytes = imgdoc.convertToPDF()
AttributeError: 'Document' object has no attribute 'convertToPDF'
```
So I suggest the version of `PyMyPDF` in `requirements.txt` and the python script be modified.

**Comment by HongYurui** [comment]
API name already solved by pull request #8 but not merged. `requirements.txt` remains to be modified.

**Comment by dylanyang17** [comment]
Thanks. PR #7 is merged.
